Assets/Tests/InputSystem/CoreTests_ProjectWideActions.cs

```diff
@@ -0,0 +1,214 @@
+#if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+
+using System;
+using System.IO;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+#if UNITY_EDITOR
+using UnityEngine.InputSystem.Editor;
+#endif
+
+internal partial class CoreTests
+{
+    const string TestCategory = "ProjectWideActions";
+    const string TestAssetPath = "Assets/TestInputManager.asset";
+    string m_TemplateAssetPath;
+
+#if UNITY_EDITOR
+        const int initialActionCount = 2;
+        const int initialMapCount = 1;
+#else
+        const int initialActionCount = 17;
+        const int initialMapCount = 2;
+#endif
+
+    [SetUp]
+    public override void Setup()
+    {
+        // @TODO: Currently we can only inject the TestActionsAsset in PlayMode tests.
+        // It would be nice to be able to inject it as a Preloaded asset into the Player tests so
```

Comment:
Should this be added as a JIRA task? Feels like we're still missing this part of testing which sounds important (but not a blocker of course).

Reply:
There is no proposed solution. It's currently not possible AFAIK.
It's an explanation of why it's done this way and a call for anyone who knows better to try to fix it.